TDAT: public void test get customer ( ) throws exception {  <NL> system . out . println ( " get customer " ) ;  <NL>  <NL> transaction bo instance = new transaction bo ( ) ;  <NL>  <NL> customer bo exp result = null ;  <NL> customer bo result = instance . get customer ( ) ;  <NL> assert equals ( exp result , result ) ;  <NL>  <NL>  / / todo review the generated test code and remove the default call to fail .  <NL> fail ( " the test case is a prototype . " ) ;  <NL>  }  COM: <s> test of get customer method of class edu </s>